dm/docs/userguide.md

```diff
@@ -370,7 +370,7 @@ Proceed as follows:
 
 ```shell
 cd dm
-sudo make prerequisites        # Installs prerequisites in system python
+sudo make cft-prerequisites        # Installs prerequisites in system python
```

Comment:
Let's keep the indenting aligned (for the comment).

Reply:
Fixed.